src/qt/coincontroldialog.cpp

```diff
@@ -421,7 +421,19 @@ void CoinControlDialog::updateLabels(CCoinControl& m_coin_control, WalletModel *
         std::vector<unsigned char> witnessprogram;
         if (out.txout.scriptPubKey.IsWitnessProgram(witnessversion, witnessprogram))
         {
-            nBytesInputs += (32 + 4 + 1 + (107 / WITNESS_SCALE_FACTOR) + 4);
+            // add input skeleton bytes (outpoint, scriptSig size, nSequence)
+            nBytesInputs += (32 + 4 + 1 + 4);
```

Comment:
bit of a nit: the outpoint is 36 [right](https://bitcoinops.org/en/tools/calc-size/)? maybe good to consolidate 32 and 4 or breakdown the _outpoint_ comment (into txid and vout index number) to prevent confusion?